vault/resource_pki_secret_backend_crl_config_test.go

```diff
@@ -63,6 +69,12 @@ resource "vault_pki_secret_backend_crl_config" "test" {
 
   expiry = "72h"
   disable = true
+  ocsp_disable = false
+  ocsp_expiry = "13h"
```

Comment:
Since we now have computed values for some fields, we need to test that functionality by not specifying them in the terraform code. Additional `resource.TestStep`s can be added to test explicit configuration changes.